### DISCUSSION TITLE
Route Overview Map

### DISCUSSION THREAD

**Discussion by AltexGames** [Traewelling/traewelling#2538] [Questions / Support] [answered]
Hello,

unfortunately I have not been able to find the answer or the right place in the code, so I am hoping for a conclusive answer here.

Is the overview map, which shows the route taken by a train, based on real data, i.e. a route API from the Deutsche Bahn etc..., or is only a "normal" map service used for getting the data.

regards
Alex

**Comment by MrKrisKrisu** [comment, marked as answer]
The routes shown are taken from Deutsche Bahn's HAFAS data. Träwelling also tries to recalculate the route with [Brouter](https://brouter.de/brouter-web/#map=9/52.4915/13.3969/standard&profile=rail) so that it is more detailed.

Unfortunately, neither is a guarantee for a correct route. Even the DB often has incorrect routes. For example, you can sometimes see ICE connections over routes without overhead lines.
